wire_test.go

```diff
@@ -206,6 +207,86 @@ func TestServerWritingResult(t *testing.T) {
 	})
 }
 
+func TestServerHandlingMultipleConnections(t *testing.T) {
+	address := TConstructMockPsqlServer(t)
+	databaseUrl := fmt.Sprintf("postgres://%s:%d", address.IP, address.Port)
+	conn, err := sql.Open("pgx", databaseUrl)
```

Comment:
Small nitpick. I would like to suggest to rename this variable to `connstr` in order to stay consistent with the other tests.

```suggestion
	connstr := fmt.Sprintf("postgres://%s:%d", address.IP, address.Port)
	conn, err := sql.Open("pgx", connstr)
```

Reply:
done